src/dune_engine/sandbox.ml

```diff
@@ -59,6 +60,7 @@ module Item = struct
     match kind with
     | S_DIR -> Directory { perms = (Path.Untracked.stat_exn path).st_perm }
     | S_REG -> File
+    | S_LNK -> Link
```

Comment:
Where is `S_LNK` coming from? Seems like only `stat_exn`s are around but we need an `lstat_exn`, right?

Reply:
It comes from listing the directory with `Path.Untracked.readdir_unsorted_with_kinds`.

Reply:
Ah, got it, thanks.